### DISCUSSION TITLE
Language of error messages while sending form

### DISCUSSION THREAD

**Discussion by nothera** [PHPMailer/PHPMailer#2463] [Q&A]
Hello, was just wondering. Is there any way, how to change the language of error messages or error messages themselves from English to some other language?

![example](https://user-images.githubusercontent.com/88513387/128421289-fbd33806-43bd-4cbc-a170-bc78b20888a2.png)

Thank you!

Tomáš

**Comment by Synchro** [comment]
While you can change the language of error messages (see the readme for how to do that), that message did not come from PHPMailer.

**Comment by nothera** [comment]
Thank you. I have realized that it changes automatically with different browser language settings.

T.
